jobs/JGDAS_GLOBAL_AERO_ANALYSIS_FINALIZE

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA=${DATA:-${DATAROOT}/${RUN}aeroanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "aeroanlfinal" -c "base aeroanl aeroanlfinal"
+
+##############################################
+# Set variables used in the script
+##############################################
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
```

Comment:
As the name of this script suggests, it will only run in `CDUMP=gdas`, correct?

Reply:
Got it, will fix these to be `JGLOBAL`...